GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -541,6 +542,45 @@ - (void)authenticateWithOptions:(GIDSignInInternalOptions *)options {
   [self addCompletionCallback:authFlow];
 }
 
+// Restores the current user with the authorization object from the keychain.
+- (void)restoreCurrentUserWithAuthState:(OIDAuthState *)authState {
+  if (!authState) {
+    return;
+  }
+
+  OIDIDToken *idToken =
+      [[OIDIDToken alloc] initWithIDTokenString:authState.lastTokenResponse.idToken];
+  GIDProfileData *profileData = [self profileDataWithIDToken:idToken];
+
+  GIDGoogleUser *user = [[GIDGoogleUser alloc] initWithAuthState:authState profileData:profileData];
+  [self setCurrentUserWithKVO:user];
+}
+
+// Generates user profile from OIDIDToken.
```

Comment:
Let's move this method to the "Helpers" section.

Reply:
Done.